include/deal.II/matrix_free/cuda_matrix_free.h

```diff
@@ -307,17 +307,20 @@ namespace CUDAWrappers
 
 
   // This function determines the number of cells per block, possibly at compile
-  // time
+  // time (by virtue of being 'constexpr')
   // TODO this function should be rewritten using meta-programming
   __host__ __device__ constexpr unsigned int
            cells_per_block_shmem(int dim, int fe_degree)
   {
-    return dim == 2 ?
-             (fe_degree == 1 ?
-                32 :
-                fe_degree == 2 ? 8 :
-                                 fe_degree == 3 ? 4 : fe_degree == 4 ? 4 : 1) :
-             dim == 3 ? (fe_degree == 1 ? 8 : fe_degree == 2 ? 2 : 1) : 1;
+    /* clang-format off */
+    return dim==2 ? (fe_degree==1 ? 32 :
+                     fe_degree==2 ? 8 :
+                     fe_degree==3 ? 4 :
+                     fe_degree==4 ? 4 :
+                     1) :
+           dim==3 ? (fe_degree==1 ? 8 :
+                     fe_degree==2 ? 2 :
+                     1) : 1;
```

Comment:
You have to `/* clang-format on */` again.

Reply:
Alternatively,
```
return dim == 2 ? (fe_degree == 1 ? 32 :      //
                     fe_degree == 2 ? 8 :     //
                       fe_degree == 3 ? 4 :   //
                         fe_degree == 4 ? 4 : //
                           1) :               //
         dim == 3 ? (fe_degree == 1 ? 8 :     //
                       fe_degree == 2 ? 2 :   //
                         1) :                 //
           1;                                 //
```
(whatever looks more appealing :wink:)

Reply:
The trailing `//` variant has one minor advantage, namely that the code block is still indented (we merely force a line break.)

Reply:
But I am fine with either version. Change, or merge as you want :-)

Reply:
The hell I'm going to do :-) Given how contentious this whole thing, I'm definitely not going to merge my own changes ;-)

Someone please tell me which way to go.

Reply:
No one feels strongly about manually fixing up indentation :wink: